src/Utils/Notifications/notifications.js

```diff
@@ -0,0 +1,70 @@
+import { toast } from "react-toastify";
+const notifyAddToWishList = (name) => {
+  toast.success(`Item ${name} added to WishList`, {
+    position: "bottom-right",
+    theme: "colored",
```

Comment:
create a base config and move cmmon props into it and extend that base config for each type to avoid duplicate lines

Reply:
eg: 
```
 base = { position: 'top', theme: 'colored', autoClose: 2000} 
 toast.success('title', { ...base, position: "bottom-right",} )
```

Reply:
Done